lib/master.js

```diff
@@ -90,7 +90,9 @@ class Master extends EventEmitter {
       this.messenger.send({ action, to: 'agent', data: this.options });
 
       // start check agent and worker status
-      this.workerManager.startCheck();
+      if (process.env.EGG_SERVER_ENV !== 'local') {
```

Comment:
`if (!this.isProduction)`

Reply:
others LGTM

Reply:
Only start check in production ? It seems like an breakchange.

Reply:
https://github.com/eggjs/egg-cluster/blob/3621081eeb7eba35f1ee97bb56c0a4d7d829e24f/lib/master.js#L537

Reply:
I found you said change to `if (!this.isProduction)` , It seems change to `if (this.isProduction)` more reasonable? Only start check while `this.isProduction` was true, Am I right?